app/representers/cms/layout_representer.rb

```diff
@@ -1,20 +1,19 @@
+# frozen_string_literal: true
+
 module CMS::LayoutRepresenter
   include ThreeScale::JSONRepresenter
 
   wraps_resource
 
   property :id
-
+  property :created_at
+  property :updated_at
   property :title
   property :system_name
-  property :content_type
-  property :handler, render_nil: true
+  property :liquid_enabled
 
-  with_options(if: ->(options) { options[:short] == false }) do |p|
-    p.property :published, render_nil: true
-    p.property :draft, render_nil: true
+  with_options(if: ->(options) { !options[:short] }) do
+    property :draft, render_nil: true
```

Comment:
This prints `null` when there's no draft. To match the XML response, it should return empty, in JSON that should be `""`

In the review comment I'm attaching a list of fields that aren't returned when they are empty, and they should be returned empty.